tests/workflows/test_cluster_plots.py

```diff
@@ -0,0 +1,42 @@
+import json
+import os
+import shutil
+import subprocess
+
+import luigi
+import pytest
+from google.cloud import storage
+
+from workflows.cluster_plots.plotting import ClusterPlotAllTasks
+
+default_spark_path = (
+    "/home/nzhon/data/processed/birdclef-2022/birdnet-embeddings-with-neighbors/v1"
```

Comment:
This still needs to get sorted out; we don't want the test to rely on any specific behavior of the environment (for example, this would fail if I ran it on my machine.

Reply:
for now I can remove the default parameter then? the unit test can take in a spark_path input I just kept this as a default in case nothing is passed in